googletest/src/gtest-port.cc

```diff
@@ -58,6 +58,10 @@
 # include <sys/procfs.h>
 #endif  // GTEST_OS_QNX
 
+#if GTEST_OS_AIX
+# include <procinfo.h>
```

Comment:
also #include <sys/types.h>, according to https://www-01.ibm.com/support/knowledgecenter/ssw_aix_71/com.ibm.aix.basetrf1/getprocs.htm